mne/selection.py

```diff
@@ -39,10 +41,14 @@ def read_selection(name, fname=None, verbose=None):
 
     Parameters
     ----------
-    name : string or list of string
+    name : str or list of str
         Name of the selection. If is a list, the selections are combined.
-    fname : string
+    fname : str
         Filename of the selection file (if None, built-in selections are used).
+    spacing : str or instance of Info
+        The spacing of channel names to return. Can be "old" for "MEG 0111"
+        or "new" for "MEG0111", or an instance of Info to infer from the
+        MEG channel names in info.
```

Comment:
I would call it info and only support this

how did you come up with this pb?

can you fix later in pipeline with a more permissive string comparison?
I think we do such things in viz code